From fb192e5feac9ed77672e4acbfaec3bdad4d7684a Mon Sep 17 00:00:00 2001
From: Thomas Huth <thuth@redhat.com>
Date: Tue, 20 Aug 2019 09:24:40 +0100
Subject: [PATCH 1/5] redhat: s390x: Rename s390-ccw-virtio-rhel8.0.0 to
 s390-ccw-virtio-rhel8.1.0

RH-Author: Thomas Huth <thuth@redhat.com>
Message-id: <20190820092441.28201-2-thuth@redhat.com>
Patchwork-id: 90077
O-Subject: [RHEL-AV-8.1.0 qemu-kvm PATCH 1/2] redhat: s390x: Rename s390-ccw-virtio-rhel8.0.0 to s390-ccw-virtio-rhel8.1.0
Bugzilla: 1693772
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: David Hildenbrand <david@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert <dgilbert@redhat.com>

Bugzilla: 1693772
Upstream: n/a

We did not ship RHEL AV 8.0 for s390x to customers, so we do not need
to maintain the s390-ccw-virtio-rhel8.0.0 machine. Rename it to
s390-ccw-virtio-rhel8.1.0 instead.

Signed-off-by: Thomas Huth <thuth@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/s390x/s390-virtio-ccw.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/hw/s390x/s390-virtio-ccw.c b/hw/s390x/s390-virtio-ccw.c
index 4d8c2ec..2f4cd14 100644
--- a/hw/s390x/s390-virtio-ccw.c
+++ b/hw/s390x/s390-virtio-ccw.c
@@ -894,23 +894,23 @@ GlobalProperty ccw_compat_rhel_7_5[] = {
 };
 const size_t ccw_compat_rhel_7_5_len = G_N_ELEMENTS(ccw_compat_rhel_7_5);
 
-static void ccw_machine_rhel800_instance_options(MachineState *machine)
+static void ccw_machine_rhel810_instance_options(MachineState *machine)
 {
 }
 
-static void ccw_machine_rhel800_class_options(MachineClass *mc)
+static void ccw_machine_rhel810_class_options(MachineClass *mc)
 {
 }
-DEFINE_CCW_MACHINE(rhel800, "rhel8.0.0", true);
+DEFINE_CCW_MACHINE(rhel810, "rhel8.1.0", true);
 
 static void ccw_machine_rhel760_instance_options(MachineState *machine)
 {
-    ccw_machine_rhel800_instance_options(machine);
+    ccw_machine_rhel810_instance_options(machine);
 }
 
 static void ccw_machine_rhel760_class_options(MachineClass *mc)
 {
-    ccw_machine_rhel800_class_options(mc);
+    ccw_machine_rhel810_class_options(mc);
     compat_props_add(mc->compat_props, hw_compat_rhel_7_6, hw_compat_rhel_7_6_len);
     compat_props_add(mc->compat_props, ccw_compat_rhel_7_6, ccw_compat_rhel_7_6_len);
 }
-- 
1.8.3.1

